UnitySDK/Assets/ML-Agents/Scripts/RewardProvider/LowLevelRewardProviderComponent.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+using UnityEngine;
+
+namespace MLAgents.RewardProvider
+{
+    public class LowLevelRewardProviderComponent : MonoBehaviour
```

Comment:
I think this should be closer to the sensor component implementation. Where there is a SensorComponent abstract mono behavior that needs to be implemented for each implementation of a Sensor.

Reply:
Done